src/main/kotlin/com/numberone/daepiro/domain/user/repository/UserRepository.kt

```diff
@@ -3,4 +3,8 @@ package com.numberone.daepiro.domain.user.repository
 import com.numberone.daepiro.domain.user.entity.UserEntity
 import org.springframework.data.jpa.repository.JpaRepository
 
-interface UserRepository : JpaRepository<UserEntity, Long>
+interface UserRepository : JpaRepository<UserEntity, Long> {
+    // todo 가독성 문제때문에 스프링데이터jpa 네이밍이 아니라 다른 방식으로 전환할지 고민
+    fun findByPasswordLoginInformation_Username(username: String): UserEntity?
```

Comment:
가독성이 문제라면 JPQL 을 사용하고, 메서드 네이밍을 도메인 용어로 짓는건 어때요?

```suggestion
    // 아래의 JPQL 은 임의로 작성함
    @Query("SELECT u FROM UserEntity u WHERE u.passwordLoginInformation.username = "username")
    fun findByUsername(@Param("username") username: String): UserEntity?
```

Reply:
좋은 생각이네요! 반영해보겠습니다!